host/sgx/create.c

```diff
@@ -374,6 +374,24 @@ static oe_result_t _add_data_pages(
 }
 
 #if !defined(OEHOSTMR)
+static const uint32_t _cpuid_leaves[OE_CPUID_LEAF_COUNT] = {0,
```

Comment:
Since this is needed by host, enclave and tests, can you move this definition out to the common/cpuid.c source file as `oe_emulated_cpuid_leaves` instead of repeating the definition that needs to be kept in sync?

Reply:
Thanks for providing the info. I thought about this when creating this PR, but not quite sure where I could place the variable.  Seems it should be more better to place under common/sgx/?

Reply:
Fixed in the latest commit by providing a cpuid.c under common/sgx folder.